### ISSUE TITLE
P1: Fix DS Config accessibility for the Settings/Dashboard/Permissions dropdown

### ISSUE THREAD

**Issue by yaelleC** [grafana/grafana#56559]
When the screen size is small, the tabs at the top of the data source configuration screen become a dropdown (see screenshot).

A11y issue: That dropdown is not keyboard accessible (you cannot tab into it)

![Image](https://user-images.githubusercontent.com/42030685/194572724-a861c792-4e6c-489f-9eaf-f3122b0e4da3.png)
